examples/README.md

```diff
@@ -0,0 +1,101 @@
+# Examples Folder
+
+This folder contains example scripts to demonstrate the usage of the Acuvity SDK in TypeScript/JavaScript. These examples provide practical use cases and instructions for integrating and using the SDK in your projects.
+
+---
+
+## Files in this Folder
+
+The JS script demonstrates:
+- How to load configurations and file paths dynamically.
+- How to initialize the Acuvity SDK and use it to run a scan on given messages or files.
+- How to handle responses from the scan and log them.
+- How to handle potential errors during the scanning process.
+
+---
+
+## Prerequisites
+
+### Node
+- Node.js installed (v16+ recommended).
+- The Acuvity SDK installed via npm in the project:
+  ```bash
+  npm install @acuvity/acuvity
+    ```
+
+### bun
+- bun installed
+- The Acuvity SDK installed via bun in the project:
+  ```bash
+  bun install @acuvity/acuvity
+    ```
+
+---
+
+## For local development, after build the local acuvity package.
```

Comment:
we don't need to explain local development in the examples readme. It's for explaining people on how to consume this module, not how to develop on it.